src/main/scala/ai/privado/entrypoint/Main.scala

```diff
@@ -25,20 +25,38 @@ package ai.privado.entrypoint
 import ai.privado.auth.AuthenticationHandler
 import ai.privado.cache.AppCache
 import ai.privado.metric.MetricHandler
+import ai.privado.utility.{StatsRecorder, TimeMetricRecordConfig}
+import io.shiftleft.utils.StatsLogger
 import org.slf4j.LoggerFactory
 
 import scala.sys.exit
 
 /** Privado Core main entry point
   */
-object Main {
+object Main extends GeneralMetadataLoggers {
   private val logger = LoggerFactory.getLogger(this.getClass)
 
   def main(args: Array[String]): Unit = {
+    val statsRecorder = StatsRecorder(fullName = true, printToConsole = true, supressSubstages = false)
```

Comment:
change this `printToConsole = false`, this should avoid printing the console logs for joern passes while generating the CPG. 

Test the change locally by running a scan with any sample repo.